views/partials/foot.jade

```diff
@@ -0,0 +1,8 @@
+if scripts && scripts.length
+	each script in scripts
+		if !script.inHead
+			script(async=script.async crossorigin=script.crossorigin defer=script.defer integrity=script.integrity src=script.src text=script.text type=script.type)
+script.
+	CKEDITOR.replaceClass = null; // Disable replacing by class
+	CKEDITOR.disableAutoInline = true;
+script. linz.loadLibraries('#{linz.get('admin path')}');
```

Comment:
@smebberson Hrmm, from what I can see this just loads the bootstrap-datetimepicker for date fields. I have done some testing and it seems that it isn't working in browsers other than chrome.

I can just remove it and fix up the datepicker in another pr? I'm thinking we just load it with the common scripts and rewrite the datepicker? What do you think?